pkg/mounter/mounter-s3fs.go

```diff
@@ -173,13 +180,35 @@ var writePassWrap = func(pwFileName string, pwFileContent string) error {
 
 func (s3fs *S3fsMounter) Unmount(target string) error {
 	klog.Info("-S3FSMounter Unmount-")
-	metaPath := path.Join(metaRoot, fmt.Sprintf("%x", sha256.Sum256([]byte(target))))
-	err := os.RemoveAll(metaPath)
+	klog.Infof("Unmount args:\n\ttarget: <%s>", target)
+
+	var metaRoot string
+
+	if mountWorker {
+		klog.Info("Worker Unmounting...")
+		metaRoot = constants.MounterConfigPathOnHost
+
+		payload := fmt.Sprintf(`{"path":"%s"}`, target)
+
+		errResponse, err := createCOSCSIMounterRequest(payload, "http://unix/api/cos/unmount")
+		klog.Info("Worker Unmounting...", errResponse)
+		if err != nil {
+			return err
+		}
+
+		removeS3FSCredFile(metaRoot, target)
```

Comment:
No need to define new variable `metaRoot` at line 189 and directly use constant here